website/src/contribute/release-guide.md

```diff
@@ -302,24 +324,28 @@ There are 2 ways to trigger a nightly build, either using automation script(reco
   
       ./beam/release/src/main/scripts/start_snapshot_build.sh
 
-* Tasks included
+* The script will:
   1. Install [hub](https://github.com/github/hub) with your agreement.
   1. Touch an empty txt file and commit changes into ```${your remote beam repo}/snapshot_build```
   1. Use hub to create a PR against apache:master, which triggers a Jenkins job to build snapshot.
   
-* Tasks you need to do manually
+* Tasks you need to do manually to __verify the SNAPSHOT build__
   1. Check whether the Jenkins job gets triggered. If not, please comment ```Run Gradle Publish``` into the generated PR.
   1. After verifying build succeeded, you need to close PR manually.
   
-#### Do all operations manually
+#### (Alternative) Do all operations manually
 
 * Find one PR against apache:master in beam.
 * Comment  ```Run Gradle Publish``` in this pull request to trigger build.
 * Verify that build succeeds.
 
 
-### Verify release branch
+**********
+
+
+## 3. Verify release branch
```

Comment:
May need @markflyhigh to rewrite this section with the recent change of the validation script and jenkins job configurations.